CK2Plus_expanded/events/000_CK2Plus_startup_events.txt.txt

```diff
@@ -0,0 +1,61 @@
+
+namespace = PlusStartup
+
+#######################################
+#
+# CK2Plus Startup Events
+#
+#######################################
+#
+# CATALOG
+#
+# 001 - 100: General events
+#
+#######################################
+
+#######################################
+#
+# 001 - 100: General events
+#
+#######################################
+#
+# INDEX
+#
+# 001: Assign local culture to barons
+#
+#######################################
+
+# Assign local culture to barons
+character_event = {
+	id = PlusStartup.001
+	
+	is_triggered_only = yes	# from on_chronicle_start, fires only for the player
+	
+	hide_window = yes
+	
+	immediate = {
+		any_character = { # NOTE WARNING this is really expensive, but only used once on startup here, so it is acceptable. Don't use it elsewhere without good reason.
```

Comment:
I really don't like using `any_character` in any place. I think, in this instance, you should use either `any_independent_ruler = { any_realm_lord = { } }` or `any_title = { limit = { tier = baron } holder_scope = { } }`

Reply:
I doubt it would have much impact. I've already tested this and it like, is not noticeable at all during startup.
But if it annoys you, sure, we can also change it to some other variant.